release/src/Main.hs

```diff
@@ -54,10 +55,26 @@ main = do
           then do
               $logInfo "Make release"
               releaseToBintray upload releaseDir (map (\(a, (_, outp)) -> (a, outp)) files)
+          else $logInfo "Make dry run of release"
+
+      -- Uploading to Maven Central
+      let mavenUploadArtifacts = filter (\a -> getMavenUpload $ artMavenUpload a) artifacts
+      uploadArtifacts <- concat <$> mapM (artifactCoords optsAllArtifacts) mavenUploadArtifacts
+          fs <- artifactCoords optsAllArtifacts a
+          pure fs
```

Comment:
This line and the one above must be deleted after applying @cocreature's suggestion.